### ISSUE TITLE
Update ED2 on HPC Vignette (follow-up)

### ISSUE THREAD

**Issue by julianpistorius** [az-digitalag/organization#328]
Follow-up issue from: https://github.com/az-digitalag/organization/issues/220

Julian's notes and progress: https://hackmd.io/@julianp-diag/H12YFAwSv

Got the HPC execution working (see notes) but I need to summarize my notes into the vignette.

**Comment by Chris-Schnaufer** [comment]
@julianpistorius What's the status of this issue?

**Comment by julianpistorius** [comment]
Good question. I think there is enough information in our notes for Kristina to complete this if necessary.

What I'm not sure is whether this issue is blocked by https://github.com/danforthcenter/sentinel-detection/issues/104 and/or https://github.com/PecanProject/pecan/issues/2540

I need to talk to @dlebauer & @KristinaRiemer about this.

**Comment by KristinaRiemer** [comment]
Action items: 
- [ ] Move Pecan documentation OSF pages to infrastructure folder in repo (Kristina)
- [ ] Create simplest use case ED2 vignette (Kristina)
- [ ] (optional) Test run through vignette (Kristina & Julian)
